### DISCUSSION TITLE
Pipe Scrape Question To a File

### DISCUSSION THREAD

**Discussion by dandguru** [danielmiessler/Fabric#1074] [Q&A]
How do you pipe a web question to file?  I'm using windows powershell (maybe that is part of the problem)

I tried:

`fabric -q "What are the best at home scar removal treatments?  Expecially, what are the best silicone gel treatments?" -o "Scar Treatments.md"`

This did nothing.

`fabric -q "What are the best at home scar removal treatments?  Expecially, what are the best silicone gel treatments?" > "Scar Treatments.md"`

This created the file, but it was not text.

`fabric -sq "What are the best at home scar removal treatments?  Expecially, what are the best silicone gel treatments?" | fabric -o scar-treatments.md`

This one output to a file, but gave it a summary.  Is there a way to pipe out the whole raw web query?

**Comment by dandguru** [comment]
I'm not sure what was going on the other day, but this worked for me today on the same system:

`fabric -q "What are the best at home scar removal treatments?  Expecially, what are the best silicone gel treatments?" > Scrape2.md`

also:

`fabric -sq "What are the best at home scar removal treatments?  Expecially, what are the best silicone gel treatments?" > Scrape.md`

**Comment by rapidarchitect** [comment]
This doesnt seem to work for me, -u works, i have a JINA_API_KEY set but i get this when i try the search above
{"data":null,"code":401,"name":"AuthenticationRequiredError","status":40103,"message":"Absence of bearer token","readableMessage":"AuthenticationRequiredError: Absence of bearer token"}

**Reply by rapidarchitect** [reply]
ok re-reunning setup and re-entering the API key seems to have fixed this
